### PR TITLE
Feature/141/no results category

### LINKED ISSUE #141: (SP: 1) Implement "Sorry, no results found" block
## Description
Implement "Sorry, no results found" block that will be displayed if there are no categories matching the search
## Design 

Image

## Acceptance Criteria
- [ ] The block should be displayed if there are no categories matching the search

### PR DESCRIPTION
**Pull request by poggy33**
Add no-results-found page in Categories Page.

### CODE REVIEW

**Review by zim89 — APPROVED** [review]

File: src/pages/categories/Categories.tsx [hunk 1]
```diff
@@ -43,6 +47,22 @@ const Categories = () => {
     setIsFetched(true)
   }
 
+  useEffect(() => {
+    if (match) {
+      const categoryArray = categoriesNamesItems.filter(
+        (category) => category === match
+      )
+      if (categoryArray.length === 0) {
+        setIsCategoryFound(true)
```
Comment (nadraharr) [inline src/pages/categories/Categories.tsx:56]: A bit confusing naming. But should work.

**Review by nadraharr — APPROVED** [review]

**Review by natamrshn — APPROVED** [review]

File: src/pages/categories/Categories.tsx [hunk 2]
```diff
@@ -43,6 +47,22 @@ const Categories = () => {
...
+      } else {
+        setIsCategoryFound(false)
+      }
+    }
+    if (!match) {
+      setIsCategoryFound(false)
+    }
+  }, [isSearchButtonClicked])
```
Comment (vlad-khrychov) [inline src/pages/categories/Categories.tsx]: `categoriesNamesItems` and `match` shoud be added to array
```suggestion
  }, [isSearchButtonClicked, categoriesNamesItems, match])
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: src/pages/categories/Categories.tsx (hunk 2 above)
Comment (poggy33) [reply src/pages/categories/Categories.tsx]: added

**Review by vlad-khrychov — APPROVED** [review]

**Comment by poggy33** [comment]
Added no-results-found page to Categories page

**Pull request merged**